full-node/sov-ethereum/src/lib.rs

```diff
@@ -174,14 +177,46 @@ pub mod experimental {
             },
         )?;
 
+        #[cfg(feature = "local")]
         rpc.register_async_method("eth_accounts", |_parameters, ethereum| async move {
             Ok::<_, ErrorObjectOwned>(ethereum.eth_rpc_config.eth_signer.signers())
         })?;
 
-        // TODO https://github.com/Sovereign-Labs/sovereign-sdk/issues/502
-        rpc.register_async_method("eth_sendTransaction", |parameters, _ethereum| async move {
-            let _data: reth_rpc_types::TransactionRequest = parameters.one().unwrap();
-            unimplemented!("eth_sendTransaction not implemented")
+        #[cfg(feature = "local")]
+        rpc.register_async_method("eth_sendTransaction", |parameters, ethereum| async move {
```

Comment:
Here is the reth code for this: https://github.com/paradigmxyz/reth/blob/main/crates/rpc/rpc/src/eth/api/transactions.rs#L439 

Apart from signing transactions it also estimates gas. The logic is quite complex so for now we can just sign the transaction and once it is working, we can implement the gas estimation in another PR.